lib/seattleflu/db/cli/command/preprocess.py

```diff
@@ -141,8 +141,9 @@ def load_uw_metadata(uw_filename: str) -> pd.DataFrame:
     Given a filename *uw_filename*, returns a pandas DataFrame containing
     clinical metadata.
     """
+    na_values = ['Unknown']
     if uw_filename.endswith('.csv'):
-        df = pd.read_csv(uw_filename)
+        df = pd.read_csv(uw_filename, na_values=na_values)
     else:
         df = pd.read_excel(uw_filename)
```

Comment:
Should `na_values` also be passed to `read_excel`?

Reply:
Nope. So long as the default na_values are kept in pandas (keep_default_na=True (by default)), 'NULL' will be converted to null.

Reply:
It gets added later just not part of this commit.